controllers/storagecluster/external_resources.go

```diff
@@ -683,6 +684,10 @@ func getTopologyConstrainedPoolsExternalMode(data map[string]string) (string, er
 	var topologyConstrainedPools []topologyConstrainedPool
 
 	domainLabel := data["topologyFailureDomainLabel"]
+	// hostname is the default domain label for host topology
```

Comment:
This looks to be a new change? @parth-gr was domainLabel host was ever tested <4.21?

Reply:
I dont remember testing it, I tested the zone based topology